### DISCUSSION TITLE
Ancient Horn Improvements

### DISCUSSION THREAD

**Discussion by Xfrtrex** [FrozenBlock/WilderWild#25] [Ideas]
Dear Lunade, it has come to my attention that the Horns of the Ancient appear rather anticlimactic and impractical, that the use of a Bow with the enchantment power five would do you much greater than that of a puny Ancient Horn, and that perhaps the Horn shall do a monstrous amount of damage to the enemy, at the cost of a quite long cooldown.

**Comment by Xfrtrex** [comment]
@AViewFromTheTop

**Reply by AViewFromTheTop** [reply]
ok
